docs/samples.md

```diff
@@ -2,17 +2,18 @@ The core of any LIMS is managing samples and ensuring integrity of their data an
 
 # Auditing
 
+TODO: I think this section needs more work, in particular with regards to the discusions we have been having about versioning. /JD 2019-05-20
```

Comment:
Are there any particular inaccuracies we can fix now? All of this documentation will need several rounds.

Reply:
For example the part "The history of samples is maintained by doing a full copy of them in the database before editing them", is something that I think is inaccurate now right? With the db design we looked at yesterday you don't need to copy the whole object on every write?

Also, I think that the part about GDPR is something that could perhaps go into a separate part of the documentation, that goes more into detail about audit tracking, data removal mechanisms, etc. But perhaps that is something that we can get back to revising later?

Reply:
I totally agree with the GDPR thing, that's a typical example of bad documentation, where one documents the thinking behind decisions when one is supposed to make it sound clear. And yeah, missed that bit about the full copy, thanks for pointing that out!

I'll fix this soon :)

Reply:
Another thing this kind of statement does is send people down an unnecessary rabbit hole they don't need to visit, e.g. "what's an append-only database [15 mins on wikipedia later]. Ok where was I?".